lib/iris/util.py

```diff
@@ -1080,6 +1080,10 @@ def new_axis(src_cube, scalar_coord=None):
     if scalar_coord is not None:
         scalar_coord = src_cube.coord(scalar_coord)
 
+    if src_cube.shape is ():
+        if isinstance(src_cube.data, np.ma.core.MaskedConstant):
+            src_cube.data = np.ma.array(data=[0], mask=[1])
```

Comment:
Can you please use `ma.array(...`, we have an `import numpy.ma as ma` in this file so we don't need the leading `np.`. This should be true everywhere in iris.